server/config_push_proxy.go

```diff
@@ -81,6 +82,16 @@ func LoadConfig(fileName string) (*ConfigPushProxy, error) {
 	if !cfg.EnableConsoleLog && !cfg.EnableFileLog {
 		cfg.EnableConsoleLog = true
 	}
+
+	// Set timeout defaults
+	if cfg.SendTimeoutSec == 0 {
+		cfg.SendTimeoutSec = 30
+	}
+
+	if cfg.RetryTimeoutSec == 0 {
+		cfg.RetryTimeoutSec = 8
+	}
+
 	if cfg.EnableFileLog {
```

Comment:
We should add a check here to verify that the retryTimeout isn't greater than the sendTimeout.